tests/test_installer.py

```diff
@@ -30,6 +31,10 @@ def test_installer_help(installer: list):
         assert long_help in result.stdout, f"Expected '{long_help}' in tool output"
 
 
+@pytest.mark.skipif(
+    Platform.current() == Platform.Linux_armv7l,
+    reason="TODO: Remove once 0.9.0 is released",
```

Comment:
I had to skip these two tests for now until 0.9.0 is released, as it tried to fetch the `armv7l`, but that is obviously not possible since I am adding it in this PR.